core/src/main/java/edu/wpi/grip/core/operations/networktables/NTManager.java

```diff
@@ -0,0 +1,64 @@
+package edu.wpi.grip.core.operations.networktables;
+
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.Singleton;
+import edu.wpi.first.wpilibj.networktables.NetworkTable;
+import edu.wpi.first.wpilibj.networktables.NetworkTablesJNI;
+import edu.wpi.grip.core.ProjectSettings;
+import edu.wpi.grip.core.events.ProjectSettingsChangedEvent;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class encapsulates the way we map various settings to the global NetworkTables state.
+ */
+@Singleton
+public class NTManager {
+
+    /**
+     * Information from:
+     * https://github.com/PeterJohnson/ntcore/blob/master/src/Log.h
+     * and
+     * https://github.com/PeterJohnson/ntcore/blob/e6054f543a6ab10aa27af6cace855da66d67ee44/include/ntcore_c.h#L39
+     */
+    private Map<Integer, Level> ntLogLevels = new HashMap<Integer, Level>() {{
```

Comment:
NetworkTables apparently uses integers to represent logging levels in a language-independent way.  We want to log using Java's built-in logger, so this map lets us take a NetworkTable level number and get a Java `Level`.